### DISCUSSION TITLE
How to hook into laragons "Start all" button?

### DISCUSSION THREAD

**Discussion by SelfMade2001** [leokhoa/laragon#992] [Q&A] [answered]
Every time I want to start my laravel app, I do not only need to press "Start all" in laragon, but I also need to execute some commands in the terminal. In my case, I want to start meilisearch.exe ( but it could be basically every command, maybe starting a laravel queue worker). Is it possible to make laragon execute some predefined terminal commands, or execute some given .exe file whenever I press the laragon "Start all" button?

**Comment by husnilkhatimi** [comment, marked as answer]
Put the command in 
usr/user.cmd
